### DISCUSSION TITLE
备份的快照和目录无法解压。

### DISCUSSION THREAD

**Discussion by magickzl** [1Panel-dev/1Panel#6286] [Q&A]
### 1Panel 版本

v1.10.13-lts

### 问题描述

备份的快照和目录无法解压，是因为备份的密码有符号的原因吗？
所有的备份都无法解压，不提示输入密码，直接提示未知文件格式。
![PixPin_2024-08-29_12-35-56](https://github.com/user-attachments/assets/103e4699-a363-4ad8-8d64-8b565a2073c6)

### 重现步骤

所有的备份都无法解压，不提示输入密码，直接提示未知文件格式。

**Comment by wanghe-fit2cloud** [comment]
1Panel 通过 OpenSSL 工具进行加解密操作。你可以在 1Panel 的主机列表中输入解压密码，检查是否能够成功解压。此外，在面板设置的快照页面进行快照恢复时，也可以输入解压密码，验证是否解压成功。

**Reply by magickzl** [reply]
感谢老哥，已用openssl解密之后，成功解压。
